ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx

```diff
@@ -237,6 +237,7 @@ export class ScenePlayerImpl extends React.Component<
         },
       ],
       sources: this.props.sceneStreams.map((s) => {
+        console.log(s.url);
```

Comment:
I'm guessing this is some accidental leftover from debugging and thus can be removed?

Reply:
@gitgiggety Good catch. Removed.